Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Annotation/AddIshAnnotation.cs

```diff
@@ -0,0 +1,365 @@
+    ///                     -PubVersion "1" `
+    ///                     -LogicalId "MYCONTENTOBJECTLOGICALID" `
+    ///                     -Version "1" `
+    ///                     -Lng "en" `
+    ///                     -Type "General" `
+    ///                     -Text "My annotation text" `
+    ///                     -Status "VANNOTATIONSTATUSUNSHARED" `
+    ///                     -Category "Comment" `
+    ///                     -Address "My annotation address" `
+    ///                     -Metadata $metadata
+    /// </code>
+    /// <para>Add annotation providing required parameters and Metadata object. Cmdlet will use the latest RevisionId for the provided content object and parameter values take precedence over metadata field values</para>
+    /// </example>
+    /// <example>
+    /// <code>
+    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/InfoShareWS/" -IshUserName "username" -IshUserPassword  "userpassword"
+    /// $metadataFilter = Set-IshMetadataFilterField -Name "VERSION" -Level Version -ValueType Value -Value "1" `
+    ///                  | Set-IshMetadataFilterField -Name "DOC-LANGUAGE" -Level Lng -ValueType Value -Value "en"
+    /// $ishObject = Get-IshDocumentObj -IshSession $ishsession -LogicalId "MYCONTENTOBJECTLOGICALID" -MetadataFilter $metadataFilter
+    ///	$metadata =	Set-IshMetadataField -Name "FISHANNOTPROPOSEDCHNGTXT" -Level Annotation -Value "My proposed change text"
+    ///	$ishAnnotation = Add-IshAnnotation -IshSession $ishsession `
+    ///                     -PubLogicalId "GUID-MYPUBLICATIONLOGICALID" `
+    ///                     -PubVersion "1" `
+    ///						-IshObject $ishObject `
+    ///                     -Type $annotationType `
+    ///                     -Text "My annotation text" `
+    ///                     -Status "VANNOTATIONSTATUSUNSHARED" `
+    ///                     -Category "Comment" `
+    ///                     -Address "My annotation address" `
+    ///						-Metadata $metadata
+    /// </code>
+    /// <para>Add annotation providing IshObject of type IshDocumentObj. Cmdlet will use the latest RevisionId for the provided content object and parameter values take precedence over metadata field values</para>
+    /// </example>
+    /// <example>
+    /// <code>
+    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/InfoShareWS/" -IshUserName "username" -IshUserPassword  "userpassword"
+    /// $ishAnnotation = Add-IshAnnotation -IshSession $ishsession `
+    ///                     -PubLogicalId "GUID-MYPUBLICATIONLOGICALID" `
+    ///                     -PubVersion "1" `
+    ///                     -LogicalId "MYCONTENTOBJECTLOGICALID" `
+    ///                     -Version "1" `
+    ///                     -Lng "en" `
+    ///                     -Type "General" `
+    ///                     -Text "My annotation text" `
+    ///                     -Status "VANNOTATIONSTATUSUNSHARED" `
```

Comment:
Might be a good explicit pester test to see if the parameter accepts the label "Private" and the element name "VANNO...UNSHARED"

Reply:
Agreed (I use VANNOTATIONSTATUSUNSHARED in tests everywhere), added test with parameter -Status "Unshared", if that is what you mean, as we don't have label "private"